src/routes/Routes.tsx

```diff
@@ -133,9 +136,11 @@ const Routes = () => {
         <Route
           path={COLONY_HOME_ROUTE}
           element={
-            <Default routeProps={{ hasBackLink: false }}>
-              <ColonyHome />
-            </Default>
+            <ColonyContextProvider>
+              <Default routeProps={{ hasBackLink: false }}>
+                <ColonyHome />
+              </Default>
+            </ColonyContextProvider>
```

Comment:
Why in here, and not inside `Entry`  nested under the `AppContextProvider` ?

That way it would wrap around, and be available to all routes

Reply:
Agree with Raul, I think it would be better to have under the `AppContextProvider`, this way all the contexts will also be grouped together (on top of not repeating code around each route)

Reply:
The only reason for putting it here is that there are some routes (not many but landing page, create colony, user pages) that don't need access to this context. 
I have refactored the context to return plain `children` if there's no `colonyName` in the path and will use it in `Entry` instead

Reply:
^ I'm taking this back. In order to access the `colonyName` param in the path, `ColonyContextProvider` needs to be inside a route that specifies such param. 
@rdig Any thoughts?

Reply:
I guess that's a good argument to have it in the route then, since we'll use route params quite a bit